app/helpers/application_helper.rb

```diff
@@ -1,2 +1,19 @@
 module ApplicationHelper
+
+  def split_and_validate_ids(input)
+    return false unless input =~ /[a-z0-9\-_\/\n]/
+    ids = input.split(/\s/).reject(&:empty?)
+    return false unless ids.all? {|id| /cpb-aacip[\-_\/][0-9]{1,3}[\-_\/][a-zA-Z0-9]/ =~ id }
```

Comment:
the regex here might accidentally match if an errant non-alphanumeric character accidentally get's added to the end of one of the ids, or if characters accidentally get added to the beginning. E.g..
```
/cpb-aacip[\-_\/][0-9]{1,3}[\-_\/][a-zA-Z0-9]/ =~ "cpb-aacip-123-45^789"
# => 0
/cpb-aacip[\-_\/][0-9]{1,3}[\-_\/][a-zA-Z0-9]/ =~ "whoops-cpb-aacip-123-45789"
# => 7
```
Using anchors to match beginning and end of string, and adding a `+` for > 1 alphanum should work...
```
/\Acpb-aacip[\-_\/][0-9]{1,3}[\-_\/][a-zA-Z0-9]+\z/ =~ "cpb-aacip-123-457^89"
# => nil
/\Acpb-aacip[\-_\/][0-9]{1,3}[\-_\/][a-zA-Z0-9]+\z/ =~ "whoops-cpb-aacip-123-457^89"
# => nil
/\Acpb-aacip[\-_\/][0-9]{1,3}[\-_\/][a-zA-Z0-9]+\z/ =~ "cpb-aacip-123-45789"
# => 0 
```